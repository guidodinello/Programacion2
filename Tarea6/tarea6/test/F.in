insertarEnAvl 10
insertarEnAvl 8
insertarEnAvl 6
arregloAConjunto 4 2 3 4 6
unionDeConjuntos 3 3 6 9
asociarEnMap 5 15.0
desasociarEnMap 5
asociarEnMap 5 25.0
desasociarEnMap 5
insertarEnCP 4 4.0
insertarEnCP 9 9.0
insertarEnCP 3 3.0
estaLlenaPila
alturaDeAvl
avlMin 5
cardinalidad
perteneceAConjunto 2
perteneceAConjunto 9
iteradorDeConjunto
existeAsociacionEnMap 5
actualizarEnCP 4 1.0
prioridad 3
prioritario
Fin
