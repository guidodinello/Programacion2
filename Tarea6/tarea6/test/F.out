1>Insertado.
2>Insertado.
3>Insertado.
4>Arreglo a conjunto con 4 elementos.
5>Union.
6>Establecida la asociación.
7>Eliminada la asociación.
8>Establecida la asociación.
9>Eliminada la asociación.
10>Se insertó el elemento.
11>Se insertó el elemento.
12>Se insertó el elemento.
13>p NO está llena.
14>Altura de avl: 2.
15>
--(12,0.00)
-(11,0.00)
--(10,0.00)
---(9,0.00)
(8,0.00)
--(7,0.00)
---(6,0.00)
-(5,0.00)
---(4,0.00)
--(3,0.00)
---(2,0.00)
----(1,0.00)
16>5.
17>Pertenece.
18>Pertenece.
19>2 3 4 6 9 
20>No existe asociacion.
21>Se actualizó el valor.
22>3.00.
23>4
24>Fin.
